docs/release_notes/0.0.39.md

```diff
@@ -2,6 +2,8 @@
 
 ## Features
 
+- Allow for service user authentication through environment variables
```

Comment:
In the rest of the okclt code base we use the wording "service account". On the other hand, in those cases we're talking about kubernetes service accounts I think. Should we change to that for consistency?

Reply:
I think this could be clarified, perhaps by simply calling it an `AWS service user`? They are different things at least, service account is a Kubernetes concept, while service user is an "us" concept. I guess?

Reply:
Argh. Explicit always win. Thanks for pointing it out